Modules/Filtering/DisplacementField/test/itkDisplacementFieldJacobianDeterminantFilterTest.cxx

```diff
@@ -118,7 +120,113 @@ TestDisplacementJacobianDeterminantValue()
   }
   else
   {
-    std::cout << "Test passed." << std::endl;
+    std::cout << "Determinant value test passed" << std::endl;
+  }
+
+  // Now test that the determinant is consistent if the voxel space is different, but the physical space is the same.
+
+  // Define physical point to test
+  itk::Point<double, 2> physPt;
+  dispacementfield->TransformIndexToPhysicalPoint(index, physPt);
```

Comment:
`dispacementfield` is clearly a typo. While fixing, also turn it into camelCase: `displacementField`.